### ISSUE TITLE
CI fails due to license update

### ISSUE THREAD

**Issue by jannyHou** [strongloop/loopback-component-explorer#164]
CI fails due to license update:
```
1) explorer with custom front-end files overrides swagger-ui files:
     Error: expected '/* custom swagger-ui file */\n' response body, got '// Copyright IBM Corp. 2014. All Rights Reserved.\n// Node module: loopback-component-explorer\n// This file is licensed under the MIT License.\n// License text available at https://opensource.org/licenses/MIT\n\n/* custom swagger-ui file */\n'
      at error (node_modules/supertest/lib/test.js:265:13)
      at Test._assertBody (node_modules/supertest/lib/test.js:191:16)
      at Test._assertFunction (node_modules/supertest/lib/test.js:247:11)
      at Test.assert (node_modules/supertest/lib/test.js:148:18)
      at Server.assert (node_modules/supertest/lib/test.js:127:12)
      at emitCloseNT (net.js:1521:8)
      at process.fallback (node_modules/loopback/node_modules/continuation-local-storage/node_modules/async-listener/index.js:450:15)
```

failure link: http://ci.strongloop.com/job/loopback-component-explorer/label=ubuntu-4/955/consoleFull

**Comment by bajtos** [comment]
@jannyHou I believe this problem was already fixed by #160, PTAL.

**Comment by jannyHou** [comment]
@bajtos thank you, true, it's fixed by https://github.com/strongloop/loopback-component-explorer/pull/160
I build master branch from jenkins and all green:
http://ci.strongloop.com/job/loopback-component-explorer/965/
The previous failure was caused by that fix not included.
I am closing this issue now.
